cmd/kube-controller-manager/app/options/nodelifecyclecontroller.go

```diff
@@ -46,7 +46,7 @@ func (o *NodeLifecycleControllerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.Int32Var(&o.LargeClusterSizeThreshold, "large-cluster-size-threshold", 50, "Number of nodes from which NodeController treats the cluster as large for the eviction logic purposes. --secondary-node-eviction-rate is implicitly overridden to 0 for clusters this size or smaller.")
 	fs.Float32Var(&o.UnhealthyZoneThreshold, "unhealthy-zone-threshold", 0.55, "Fraction of Nodes in a zone which needs to be not Ready (minimum 3) for zone to be treated as unhealthy. ")
 	fs.BoolVar(&o.EnableTaintManager, "enable-taint-manager", o.EnableTaintManager, "WARNING: Beta feature. If set to true enables NoExecute Taints and will evict all not-tolerating Pod running on Nodes tainted with this kind of Taints.")
-	fs.StringSliceVar(&o.TenantPartitionKubeConfigs, "tenant-server-kubeconfigs", o.TenantPartitionKubeConfigs, "Comma separated string representing tenant api-server kubeconfigs.")
+	fs.StringVar(&o.TenantPartitionKubeConfig, "tenant-server-kubeconfig", o.TenantPartitionKubeConfig, "Comma separated string representing tenant api-server kubeconfig(s).")
```

Comment:
is there a reason to use stringvar (an later manual parsing into slice) instead of stringslicevar?

Reply:
Currently we support pass in multiple kubeconfig files. In next generation, with the support of auto detect new/remove tenant partitions, there should be only one kubeconfig. Prefer not to update interface frequently due to backwards compatibility concerns.